README.md

```diff
@@ -150,7 +140,6 @@ $ npm start
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [Issues](https://github.com/Bigizi/Space-Travel-Hub/issues).
```

Comment:
- [ ] Also, in your `ACKNOWLEDGEMENTS` section. I only can see credits for Microverse. But I didn't find the given credits for the creator of the design. Kindly give credit to the designer.

> Requirement: The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).